service/remote_test.go

```diff
@@ -68,6 +69,12 @@ func (m *MyComp) RemoteErr(ctx context.Context) (*test.SomeStruct, error) {
 
 type unregisteredStruct struct{}
 
+func errorIs(t *testing.T, err1 error, err2 error)  {
+	if !assert.ObjectsAreEqual(err1, err2) && !errors.Is(err1, err2) {
```

Comment:
Shouldn't it be `!assert.ObjectsAreEqual(err1, err2) || !errors.Is(err1, err2)`?

Reply:
I think we assume that the struct being equal is not enough to consider it as the same (same for the opposite). But @victor-carvalho has more context on it.

Reply:
Gotcha. Just double checking, anyway 😂